model/event/subscribers.go

```diff
@@ -99,6 +104,11 @@ func (s *Subscriber) String() string {
 	case *GithubMergeSubscriber:
 		subscriberStr = v.String()
 
+	case CommitQueueDequeueSubscriber:
```

Comment:
all of these cases except *string and *string can be reduced to "fmt.Stringer"

Reply:
done.